.github/actions/static-analysis-tests/action.yml

```diff
@@ -0,0 +1,52 @@
+name: Terraform Static Analysis Tests
+description: Runs Terraform Static Analysis Tests - format, validate and plan
+inputs:
+  GITHUB_TOKEN:
+    description: Workflow GitHub Token (limited permissions)
+    required: true
```

Comment:
🔧  If the `github-token` on line 23 is removed, this can be removed as well.
```suggestion

```

Reply:
removed!